src/main/java/xyz/nucleoid/dungeons/dungeons/util/item/material/DgArmorMaterial.java

```diff
@@ -1,31 +1,196 @@
 package xyz.nucleoid.dungeons.dungeons.util.item.material;
 
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.item.*;
+import net.minecraft.recipe.Ingredient;
+import net.minecraft.sound.SoundEvent;
+import xyz.nucleoid.dungeons.dungeons.item.armor.DgArmorType;
 import xyz.nucleoid.dungeons.dungeons.util.item.DgItemQuality;
 
-public enum DgArmorMaterial implements DgMaterial {
+import java.util.Random;
 
-    // Tier 1
-    LEATHER("leather"),
+public enum DgArmorMaterial implements DgMaterial, ArmorMaterial {
+    // Light Armor
+    CLOTH("cloth", 1F, 0F, DgArmorType.LIGHT_ARMOR, DgItemQuality.BATTERED, DgItemQuality.FINE, ArmorMaterials.LEATHER, 14609387),  //#deebeb
+    IRON_THREAD("iron_thread", 2F, 0F, DgArmorType.LIGHT_ARMOR, DgItemQuality.MEDIOCRE, DgItemQuality.SUPERB, ArmorMaterials.LEATHER, 13751765), //#d1d5d5
+    SILK("silk", 1.5F, 0F, DgArmorType.LIGHT_ARMOR, DgItemQuality.MEDIOCRE, DgItemQuality.LEGENDARY, ArmorMaterials.LEATHER, 16448205), //#fafacd
+    // Medium Armor
+    LEATHER("leather", 4F, 0F, DgArmorType.MEDIUM_ARMOR, DgItemQuality.BATTERED, DgItemQuality.FINE, ArmorMaterials.LEATHER, 10511680), //Default
+    CHAINS("chains", 6F, 0F, DgArmorType.MEDIUM_ARMOR, DgItemQuality.MEDIOCRE, DgItemQuality.SUPERB, ArmorMaterials.CHAIN, -1),
+    HIDE("hide", 3F, 0F, DgArmorType.MEDIUM_ARMOR, DgItemQuality.MEDIOCRE, DgItemQuality.LEGENDARY, ArmorMaterials.LEATHER, 13467442), // #cd7f32
+    // Heavy Armor
+    IRON_PLATING("iron_plate", 10F, 2F, DgArmorType.HEAVY_ARMOR, DgItemQuality.BATTERED, DgItemQuality.FINE, ArmorMaterials.IRON, -1),
+    DIAMOND_PLATING("diamond_plate", 12F, 2F, DgArmorType.HEAVY_ARMOR, DgItemQuality.MEDIOCRE, DgItemQuality.SUPERB, ArmorMaterials.DIAMOND, -1),
+    NETHERITE_PLATING("netherite_plate", 15F, 3F, DgArmorType.HEAVY_ARMOR, DgItemQuality.MEDIOCRE, DgItemQuality.LEGENDARY, ArmorMaterials.NETHERITE, -1),
     ;
```

Comment:
Move to L24 maybe

Reply:
The semicolon? If that's what you meant, I'll fix it